### DISCUSSION TITLE
[Bug]: Using 'Submit frame to Frigate+' while in full screen causes dialog in background, appears frozen

### DISCUSSION THREAD

**Discussion by YBonline** [blakeblackshear/frigate#12303] [Report a Bug] [answered]
### Describe the problem you are having

When in full screen, pressing the submit frame to frigate+ causes the dialog confirming the submission to open in the background with focus, but the video frame is still in front of it, so you can't see the dialog. Nothing responds in the full screen window when this occurs, because the dialog having focus is not visible. Exiting full screen reveals the dialog and allows continued normal operation.

### Steps to reproduce

na

### Version

0.14.0-bfbacee

### Frigate config file

```yaml
NA
```

### Relevant log output

```shell
na
```

### Operating system

Debian

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

na

**Comment by hawkeye217** [comment, marked as answer]
Fixed in https://github.com/blakeblackshear/frigate/pull/12297
